src/components/extractFields.js

```diff
@@ -141,7 +141,6 @@ function getMultiLangNames(defaultName, properties) {
       return {
         key: key.substring(key.indexOf(':') + 1, key.indexOf(':') + 4), // get the iso part of the key name:iso_x_preferred
         value: properties[key]
-                .filter(name => !defaultName || defaultName.indexOf(name) < 0) // remove duplicate elements found in default name
```

Comment:
The [pelias/model](https://github.com/pelias/model) library we use as a common system for describing Pelias documents now enforces this filter, so this code change won't have the effect you desire.

In general, we've found that duplicating a value in `name.default` and `name.${lang}` leads to problems. So we would like to ensure that those duplicates do **not** exist, while it seems like the intention of your change here is to enforce that they **do** exist.

Can you give some examples of queries that aren't behaving how you'd like that lead you to make this change? Please use our [Compare tool](https://pelias.github.io/compare/) if you can so we have a common environment to discuss.